packages/react-percy-webpack/src/entry/index.js

```diff
@@ -1,21 +1,14 @@
+import findSnapshotFiles from './findSnapshotFiles';
 import merge from 'webpack-merge';
-import path from 'path';
-import VirtualModulePlugin from 'virtual-module-webpack-plugin';
 
-export default function configureVirtualEntry(webpackConfig, percyConfig, entry) {
-  const virtualEntryPath = path.join(__dirname, 'percy-virtual-entry.js');
+export default function configureEntry(webpackConfig, percyConfig) {
+  const snapshotFiles = findSnapshotFiles(percyConfig);
 
   return merge.strategy({
     entry: 'replace',
   })(webpackConfig, {
     entry: {
-      percy: [...percyConfig.includeFiles, virtualEntryPath],
+      percy: [...percyConfig.includeFiles, ...snapshotFiles],
```

Comment:
For right now we're still just generating a single entry file. As a next step, I'm going to work on splitting out each snapshot file into its own entry so that we can actually wrap each file in a suite. As of right now, all we can do is wrap this entire single file in a suite.